qpc/utils.py

```diff
@@ -340,18 +340,14 @@ def setup_logging(verbosity):
     else:
         log_level = logging.DEBUG
 
+    log_fmt = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
     # Using basicConfig here means that all log messages, even
     # those not coming from qpc, will go to the log file
-    logging.basicConfig(filename=QPC_LOG, format='%(asctime)s - %(name)s - '
-                                                 '%(levelname)s - %(message)s')
-    # but we only adjust the log level for the 'qpc' logger.
-    log.setLevel(log_level)
-    # the StreamHandler sends warnings and above to stdout, but
-    # only for messages going to the 'qpc' logger, i.e. QPC
-    # output.
-    stderr_handler = logging.StreamHandler()
-    stderr_handler.setLevel(logging.ERROR)
-    log.addHandler(stderr_handler)
+    logging.basicConfig(filename=QPC_LOG, format=log_fmt, level=log_level)
+    stream_handler = logging.StreamHandler()
```

Comment:
What do you think about making it a separate function?

```
def get_console_handler(log_fmt, log_level):
       setup ...
        return stream_handler
```

Reply:
Mmm makes sense but I think this is small enough to justify not adding the "bureaucracy" of an extra function XD

Reply:
Fair enough =)